.env.example

```
@@ -1,19 +1,40 @@
+# ====================================
+# APPLICATION CONFIGURATION
+# ====================================
 NEXT_PUBLIC_BASE_URL="your_site_url"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)